src/components/Info/_info.scss

```diff
@@ -0,0 +1,50 @@
+.info-card__heading {
```

Comment:
This should be .info-section__heading

Reply:
Previously ,it was info-section__heading only,but after your comment that "this should be info-card__heading" I changed it,now again you are saying it should be info-section_heading.I am, really getting confused and not getting idea what is wrong here.

Reply:
Apologies but I am trying to find the problem in your commit. Since you started from your previous branch from step 3 instead of switching to step 4 per the instructions (https://www.carbondesignsystem.com/tutorial/react/step-4/#fork-clone-and-branch), your updates are not isolated as they should be. 

Please review your changes against the tutorial instructions: https://www.carbondesignsystem.com/tutorial/react/step-4/

You should also be comparing your changes when you run the application locally and once you submit your PR, you can review the build triggered by your PR: https://deploy-preview-2579--carbon-tutorial.netlify.com/